widget/entry_test.go

```diff
@@ -3211,3 +3247,7 @@ func teardownImageTest(w fyne.Window) {
 	w.Close()
 	test.NewApp()
 }
+
+func waitForBinding() {
+	time.Sleep(time.Millisecond * 100) // data resolves on background thread
+}
```

Comment:
I noticed this is duplicated in the `widget` package, shouldn't all binding tests be in `widget_test`?

Reply:
Many of the widgets did not yet have widget_test variants.
The exact setup of testing is still not finalised in our document so I didn't want to introduce the change in this PR.
We certainly could get it clarified and applied later.